docs/publishing/advanced_publishing_options.md

```diff
@@ -108,16 +110,34 @@
     
     ![Calendar widget toolbar](img/selection_action_bar.png "Calendar widget toolbar")
     
-    #### Rescheduling or cancelling events
+    ## Rescheduling or cancelling publications
     
-    In case of publishable events (e.g. articles), you can change or cancel their planned publication by using the toolbar buttons.
+    In case of publishable Content items (e.g. articles), you can change or cancel their planned publication using the **Reschedule** or **Unschedule** buttons.
+    These buttons are accessible in the [Dashboard](#rescheduling-or-cancelling-in-dashboard) and the [Calendar widget](#rescheduling-or-cancelling-with-calendar-toolbar).
     
     |Button|Description|
     |------|-----------|
     |![Reschedule button](img/selection_action_bar_reschedule.png)|Reschedule all selected events.|
     |![Discard button](img/selection_action_bar_discard.png)|Cancel the future publication of selected events.|
     
-    Select all events you want to perform a bulk action on.
+    #### Rescheduling or cancelling with Dashboard
+    
+    To reschedule or cancel events in the Dashboard, perform the following actions:
+    
+    1. Open the Dashboard by clicking the eZ logo in the left-upper corner.
+    1. In the **Me** panel, view all your scheduled Content items by clicking **My scheduled**.
+    1. From **My Publications scheduled**, select all Content items to have its publication time rescheduled or cancelled.
+    1. Using the toolbar buttons, perform one of the following actions:
+        - To change the publication time, click the **Reschedule** button.
+          In the **Reschedule publication** modal window, select the new date and click **Confirm reschedule**.
+        - To cancel publication, click the **Unschedule** button.
+          In the modal window, confirm the cancellation by clicking **Discard scheduling**.
+          
+    ![Reschedule or cancel in Dashboard](img/reschedule_cancel_dashboard.png "Rescheduling or cancelling in the Dashboard")
```

Comment:
The buttons are specified in the table, there is no need for a cursor in the screenshot.